test/test_type_promotion.py

```diff
@@ -686,6 +691,16 @@ def test_integer_div_deprecated(self, device, dtype):
         with self.maybeWarnsRegex(UserWarning, '^Integer division.+is deprecated.+'):
             torch.addcdiv(a, b, b, out=o)
 
+    @unittest.skipIf(not TEST_NUMPY, "NumPy not found")
+    @float_double_default_dtype
+    @onlyOnCPUAndCUDA
+    def test_numpy_scalar_interactions(self, device):
```

Comment:
Do you think it's better to add `dtype` to the parameter list?

Reply:
If the test becomes more exhaustive then yes, it would make sense to generate the dtype variants.